web/ui/react-app/src/pages/targets/__testdata__/testdata.ts

```diff
@@ -0,0 +1,220 @@
+/* eslint @typescript-eslint/camelcase: 0 */
```

Comment:
This file exports mutable objects which can be modified at some point from somewhere. Maybe you should export freezed Objects instead.

Reply:
Good point. It's just test data, but freezing is a good practice

Reply:
frozen